### PR TITLE
use pm2 to start durian.js

### PR DESCRIPTION
**Pull request by mario-huang**
pm2-beta seams support bun now, I want to try. If it works, then all bun apps can use it.

### CODE REVIEW

**Comment by waghanza** [comment]
Does this version of pm2 can use deno also ?

**Comment by mario-huang** [comment]
> Does this version of pm2 can use deno also ?

no, but deno has alternative to pm2 - Pup
https://deno.land/x/pup

**Comment by mario-huang** [comment]
need help:

Run bundle exec rake config
rake aborted!
NoMethodError: undefined method `unshift' for an instance of String (NoMethodError)

        value.unshift(*value2)
             ^^^^^^^^
/home/runner/work/web-frameworks/web-frameworks/Rakefile:80:in `override_or_merge'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:69:in `block in custom_config'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:66:in `each'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:66:in `custom_config'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:225:in `block ([4](https://github.com/the-benchmarker/web-frameworks/actions/runs/12480482397/job/34831395013?pr=8072#step:6:5) levels) in <top (required)>'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:224:in `each'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:224:in `block (3 levels) in <top (required)>'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:223:in `each'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:223:in `block (2 levels) in <top (required)>'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:217:in `each'
/home/runner/work/web-frameworks/web-frameworks/Rakefile:217:in `block in <top (required)>'
/home/runner/work/web-frameworks/web-frameworks/vendor/bundle/ruby/3.3.0/gems/rake-13.2.1/exe/rake:27:in `<top (required)>'
/opt/hostedtoolcache/Ruby/3.3.6/x64/bin/bundle:2[5](https://github.com/the-benchmarker/web-frameworks/actions/runs/12480482397/job/34831395013?pr=8072#step:6:6):in `load'
/opt/hostedtoolcache/Ruby/3.3.6/x64/bin/bundle:25:in `<main>'
Tasks: TOP => config
(See full trace by running task with --trace)
Error: Process completed with exit code 1.

**Comment by waghanza** [comment]
Will be great to have pup then for deno and pm2 for both bun and node

**Comment by waghanza** [comment]
Last time I've check pup was not production-ready

File: javascript/durian.js-fastify/config.yaml [hunk 1]
```diff
@@ -5,3 +5,5 @@ framework:
     - bun
   bootstrap:
     - bun install
+  command:
+    - bun run pm2
```
Comment (waghanza) [inline javascript/durian.js-fastify/config.yaml:9]: This should be a string, not an array.

Also this should be set in javascript/config.yaml file

**Review by waghanza — CHANGES_REQUESTED** [review]
Suggestion made will fix CI 😇

Could you also remove bootstrap option in durians config file, and change the one in javascript/config.yaml to bun install --production, in order to make sure Dev dependencies are not use (and to compute build time, in the future)

**Comment by mario-huang** [comment]
> Suggestion made will fix CI 😇
> 
> Could you also remove bootstrap option in durians config file, and change the one in javascript/config.yaml to bun install --production, in order to make sure Dev dependencies are not use (and to compute build time, in the future)

done that~

**Pull request closed without merging**